Public/Src/FrontEnd/TypeScript.Net/TypeScript.Net.UnitTests/TypeChecking/SwtichExpressionCheckerTests.cs

```diff
@@ -127,7 +127,37 @@ public void NoClauses()
                 @"
 let result = 'a' switch {};
 ";
-            ExpectOneError(code, "Switch expression must have at least one clause.");
+            ExpectOneError(code, "Switch expression must have at least one clause");
+        }
+
+        [Fact]
+        public void OnlyDefaultClause()
+        {
+            string code =
+                @"
+let result = 'a' switch {default: 10};
+";
+            ExpectOneError(code, "Switch expression must have at least one clause that is not 'default'.");

```

Comment:
fine with this, but I don't think allowing this case will harm

Reply:
better stricter and relax later than the inverse :)

---
In reply to: [274186597](https://github.com/Microsoft/BuildXL/pull/123#discussion_r274186597) [](ancestors = 274186597)